Apps/phvxstream/display_report.html

```diff
@@ -0,0 +1,260 @@
+{% extends 'widgets/widget_template.html' %}
+{% load custom_template %}
+
+{% block title1 %}{{ title1 }}{% endblock %}
+{% block title2 %}{{ title2 }}{% endblock %}
+{% block custom_tools %}
+{% endblock %}
+
+{% block widget_content %} <!-- Main Start Block -->
+
+<!-- 
+File: display_report.html
```

Comment:
As discussed, the copyright is generated from running this script: https://cd.splunkdev.com/phantom/appscript/-/blob/github/update_cr.py